docs/v3-migration-doc.md

```diff
@@ -89,6 +89,8 @@ Upgrade of kube-prometheus-stack is a breaking change and requires manual steps:
   Error: UPGRADE FAILED: cannot patch "collection-kube-state-metrics" with kind Deployment: Deployment.apps "collection-kube-state-metrics" is invalid: spec.selector: Invalid value: v1.LabelSelector{MatchLabels:map[string]string{"app.kubernetes.io/instance":"collection", "app.kubernetes.io/name":"kube-state-metrics"}, MatchExpressions:[]v1.LabelSelectorRequirement(nil)}: field is immutable
   ```
 
+- In case of overriding `image.repository`, please follow the `kube-prometheus-stack` [migration doc][kube-prometheus-stack-image-migration] on that.
```

Comment:
Not sure if this should be covered by migration script. I will just print information that the value has been changed

Reply:
Not sure if this description is enough for users to understand what properties are affected.

Reply:
Do you have idea how to call it better?

`kube-prometheus-stack.**.repository` ?

Reply:
Hm I would probably go with something like
> In case of overriding any of the `image.repository` properties under the `kube-prometheus-stack` property, please follow...

but I'll leave it to you to decide.